[2024-12-17T11:13:47.586+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-17T11:13:47.593+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: call_9851100067_2024-12-17T11-11-36-002772Z.branch_on_call_status scheduled__2024-12-17T11:11:36.002772+00:00 [queued]>
[2024-12-17T11:13:47.597+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: call_9851100067_2024-12-17T11-11-36-002772Z.branch_on_call_status scheduled__2024-12-17T11:11:36.002772+00:00 [queued]>
[2024-12-17T11:13:47.597+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2024-12-17T11:13:47.602+0000] {taskinstance.py:2889} INFO - Executing <Task(_BranchPythonDecoratedOperator): branch_on_call_status> on 2024-12-17 11:11:36.002772+00:00
[2024-12-17T11:13:47.607+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=140443) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-17T11:13:47.607+0000] {standard_task_runner.py:72} INFO - Started process 140444 to run task
[2024-12-17T11:13:47.608+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'call_9851100067_2024-12-17T11-11-36-002772Z', 'branch_on_call_status', 'scheduled__2024-12-17T11:11:36.002772+00:00', '--job-id', '682', '--raw', '--subdir', 'DAGS_FOLDER/complete_dag.py', '--cfg-path', '/tmp/tmp0jil75gh']
[2024-12-17T11:13:47.608+0000] {standard_task_runner.py:105} INFO - Job 682: Subtask branch_on_call_status
[2024-12-17T11:13:47.628+0000] {task_command.py:467} INFO - Running <TaskInstance: call_9851100067_2024-12-17T11-11-36-002772Z.branch_on_call_status scheduled__2024-12-17T11:11:36.002772+00:00 [running]> on host 1868de310944
[2024-12-17T11:13:47.674+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='call_9851100067_2024-12-17T11-11-36-002772Z' AIRFLOW_CTX_TASK_ID='branch_on_call_status' AIRFLOW_CTX_EXECUTION_DATE='2024-12-17T11:11:36.002772+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-17T11:11:36.002772+00:00'
[2024-12-17T11:13:47.674+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-17T11:13:47.676+0000] {python.py:240} INFO - Done. Returned value was: find_reason_for_unanswered
[2024-12-17T11:13:47.676+0000] {branch.py:38} INFO - Branch into find_reason_for_unanswered
[2024-12-17T11:13:47.676+0000] {skipmixin.py:230} INFO - Following branch ('find_reason_for_unanswered',)
[2024-12-17T11:13:47.682+0000] {skipmixin.py:278} INFO - Skipping tasks [('find_if_proper_interaction_is_completed', -1)]
[2024-12-17T11:13:47.697+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-17T11:13:47.698+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=call_9851100067_2024-12-17T11-11-36-002772Z, task_id=branch_on_call_status, run_id=scheduled__2024-12-17T11:11:36.002772+00:00, execution_date=20241217T111136, start_date=20241217T111347, end_date=20241217T111347
[2024-12-17T11:13:47.740+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-17T11:13:47.756+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-17T11:13:47.757+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
